app/helpers/form_helper.rb

```diff
@@ -94,4 +94,8 @@ def international_phone_codes_data(code_data)
       country_name: code_data['name'],
     }
   end
+
+  def idp_simple_form_for(record, options = {}, &block)
```

Comment:
WDYT of calling the method something that implies it does validation? ex `idp_validated_simple_form_for` or `validated_simple_form_for`?

Reply:
> WDYT of calling the method something that implies it does validation? ex `idp_validated_simple_form_for` or `validated_simple_form_for`?

Yeah, I think that'd work better. Honestly, any unique name could be fine. I think originally I'd thought there might have already been some validation built in with `simple_form_for`, hence it might be ambiguous to name it as such. But I guess aside from the ability to mark fields as required or use other browser validations, there's nothing really built-in.

I'll update it 👍

Reply:
> WDYT of calling the method something that implies it does validation? ex `idp_validated_simple_form_for` or `validated_simple_form_for`?

Updated in f5d7572.